docker-compose.yaml

```diff
@@ -19,11 +19,17 @@ services:
       APIM_AUTHORIZATION_URL: ${APIM_AUTHORIZATION_URL}
       APIM_HOST_EXTERNAL: ${APIM_HOST_EXTERNAL}
       APIM_HOST_INTERNAL: ${APIM_HOST_INTERNAL}
-      REDIS_HOST: ffc-sfd-data-redis
-      COSMOS_ENDPOINT: ${COSMOS_ENDPOINT}
+      COSMOS_ENDPOINT: ${COSMOS_ENDPOINT:-https://cosmos-db:8081}
```

Comment:
Suggest maybe renaming it from `cosmos-db` to `ffc-sfd-data-cosmos-db` to be consistent with others.

Also less of an issue on ADP than FCP, but will avoid collisions with other containers where we're fixing the name